panther_analysis_tool/main.py

```diff
@@ -179,17 +180,22 @@ def load_module(filename: str) -> Tuple[Any, Any]:
     return module, None
 
 
-def load_analysis_specs(directories: List[str]) -> Iterator[Tuple[str, str, Any, Any]]:
+def load_analysis_specs(directories: List[str],
+                        ignored_files: OPT[list] = None) ->\
+        Iterator[Tuple[str, str, Any, Any]]:
```

Comment:
did `make fmt` do this? I thought `black` didn't like multilines with `\` 
```suggestion
                        ignored_files: OPT[list] = None
                        ) -> Iterator[Tuple[str, str, Any, Any]]:
```

Reply:
Nah this was PyCharm being "helpful"  I need to play with the settings at some point to see if it can use black's standards instead of stuff like this. 

Fixed with a `make fmt`